tests/scripts/task_python_docs.sh

```diff
@@ -40,7 +40,9 @@ rm -rf docs/doxygen
 
 # prepare auto scheduler tutorials
 rm -rf tutorials/auto_scheduler/*.json
+rm -rf tutorials/get_started/*.json
 cp -f tutorials/auto_scheduler/ci_logs/*.json tutorials/auto_scheduler
+cp -f tutorials/auto_scheduler/ci_logs/*.json tutorials/get_started
```

Comment:
why this line?

Reply:
the particular flaky error was tune matmul in get started, the tutorial get moved over there, so we need some log to help prevent flaky issue

Reply:
in tune_matmul, it produces its own log file, no? that's why the target with new default option caused troubles in CI.

Reply:
it is for the case where you get too few rounds but no valid record i believe